### ISSUE TITLE
Add femboy crawler 

### ISSUE THREAD

**Issue by sussyamogusimposter** [Anuken/Mindustry-Suggestions#3824]
### Describe the content or mechanics you are proposing.

Add femboy crawler 

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Add femboy crawler 

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by sussyamogusimposter** [comment]
Add femboy crawler

**Comment by gorodmi** [comment]
what

**Comment by Mnemotechnician** [comment]
finally, a suggestion we all can fully agree on! 

femboy crawler for life

**Comment by RFAC-Productions** [comment]
Add Femboy Crawler

**Comment by OnyxFinger** [comment]
Disclaimer : The few early steps won't necessarily work because 69420 lol and they can be avoided by launching the game.

__**How to best get blocked on Anuken/Suggestions GitHub :**__

Guys, today I'm going to show you how to BEST get blocked on Anuken/Suggestions for any Linux user, it is the fastest and most reliable trick out there for you to get banned. Now let's get started ! 

**Part 1 :**

First what you're gonna need to do is open a suggestion and for that you will do the following steps : 
Make sure the game is a .jar file because I suck and idk how to code so I'm just gonna copypaste some stuff (I assume it can be changed to any type of file very easily but I am not going to try just a for a joke). Once it is done, place it in a folder called "Mindustry"
Then in the same folder create a script shell file via the built-in text editor, that ways exactly this : 

#!/bin/sh
cd /home/**>insert name here<**/Desk/Mindustry
java -jar Mindustry.jar

Make sure the file is a .sh guys, and make sure every word was spelled perfectly, else this trick won't work you guys !
Then, go in your "Mindustry" folder, right click and "open in a terminal"
Then you are going to type in "./Mindustry.sh" and enter.
This should launch the game. Once you've done that, you will go to DATABASE > ABOUT > SUGGESTIONS.

**Part 2 :**

You will be redirected to selection between two tags : 

![Capture d’écran de 2022-08-17 14-06-31](https://user-images.githubusercontent.com/68511236/185114429-377a6e6d-fe43-4886-9ac9-96fbd896ea20.png)

Then you must select a tag, here we will go for "balancing and feedback" you guys because our objective is to be an absolute troll and were are going to suggest instead of giving feedback, that should get us blocked nicely.
You are gonna find a template that should look like this :

![Capture d’écran de 2022-08-17 14-08-18](https://user-images.githubusercontent.com/68511236/185114807-5a061ce7-8ac5-4cae-b4b0-1a94a9e4e5f6.png)

Now you will modify the title but we want it to be infuriating, so we will make to keep the word "title" and then add the text "aha 69420" without leaving space between the "aha" and the "title"

Then for the "Describe what you would like changed, and why." you will ask for "the addition of Anuken/69 to the block list", it doesn't mean anything but that's what we want anyway.

For " Describe the changes you want to propose. Include possible alternatives." say that the Anuken/69 allows you to create your own technology via an in-game modding tool that can read just about anything in somebody's PC and connect to the internet, no worry this isn't shady at all, it's just an ancient technique developped by a secret alliance between the French and the British.

Then I want you to re-do **part 1**
This time, instead of choosing "Balancing and feedback" you will choose "Feature request". You will then be presented by this : 

![Capture d’écran de 2022-08-17 14-16-20](https://user-images.githubusercontent.com/68511236/185116247-6a89ef4c-84d6-4408-a1d0-2251a502ceb9.png)

You will not write anything there, just copy this : 
Before making this issue, check the boxes below to confirm that you have acknowledged them.
I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version. *
I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet. *
I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category. *

Then you will paste it in your balancing/feedback suggestion, and re-add the boxes via "add task" that should be just above the selected typing area.

![Capture d’écran de 2022-08-17 14-19-45](https://user-images.githubusercontent.com/68511236/185116949-11cc3f77-9969-437e-a2f6-adc34f294ab1.png)

Then, you will NOT check the existing boxes,and you will add a new one called "I want to get banned", that you will check.

After that, click on "Submit new issue" and you will be banned by dawn.

I hope you guys enjoyed this guide, I had a ton of fun wasting an hour of my life writing a paragraph and coming up with new tricks to facilitate your task, such as opening the game from a console and getting to the link there, and I hope you enjoy your block while it lasts, because statistics have shown that you are fatherless.

**Comment by Mnemotechnician** [comment]
> [comment containing a wall of text with pictures](https://github.com/Anuken/Mindustry-Suggestions/issues/3824#issuecomment-1217939035)

You should post this on https://gist.github.com and not here lol

**Comment by OnyxFinger** [comment]
> > [comment containing a wall of text with pictures](https://github.com/Anuken/Mindustry-Suggestions/issues/3824#issuecomment-1217939035)
> 
> You should post this on https://gist.github.com and not here lol

I was trolling.

**Comment by Mnemotechnician** [comment]
> I was trolling.

(you clearly don't understand what trolling means)

**Comment by OnyxFinger** [comment]
> > I was trolling.
> 
> (you clearly don't understand what trolling means)

then educate me

**Comment by Mnemotechnician** [comment]
> then educate me

https://dictionary.cambridge.org/dictionary/english/trolling
https://www.urbandictionary.com/define.php?term=Trolling

**Comment by OnyxFinger** [comment]
> > then educate me
> 
> https://dictionary.cambridge.org/dictionary/english/trolling https://www.urbandictionary.com/define.php?term=Trolling



> Trolling – (verb), as it relates to internet, is the deliberate act, (by a Troll – noun or adjective), of making random unsolicited and/or controversial comments on various internet forums with the intent to provoke an emotional [knee jerk](https://www.urbandictionary.com/define.php?term=knee%20jerk) reaction from unsuspecting readers to engage in a fight or argument

My message was a deliberate act of making a random and unsolicitated comment on GitHub, and provoked a reaction from an ensuspecting reader who engaged in an argument. Pretty close. Plus, one could argue that it would make some people cringe out when seeing my poor coding skills, considering that many active people on the suggestion/bug/issue repositories are coders themselves, or people with knowledge of it.

**Comment by Mnemotechnician** [comment]
*as someone somewhen said, if you end up explaining your trolling, it's a bad trolling*

**Comment by OnyxFinger** [comment]
> _as someone somewhen said, if you end up explaining your trolling, it's a bad trolling_

-> Trolling targets individuals who don't understand it is trolling.

**Comment by Mnemotechnician** [comment]
(if it isn't clear, the first message was a joke, i thought you were just playing along...)

ao, anyways, how 'bout we all agree that mindustry really needs a femboy crawler unit

**Comment by OnyxFinger** [comment]
> (if it isn't clear, the first message was a joke, i thought you were just playing along...)

oh xD
I wasn't sure

> ao, anyways, how 'bout we all agree that mindustry really needs a femboy crawler unit

Agreed brother ~~wait fuck~~
